doc/rfcs/0002-support-run-component-without-tiup.md

```diff
@@ -0,0 +1,23 @@
+# Support Run Component without TiUP
+
+## Summary
+
+Allow users to only use TiUP during installation and upgrade, without needing to invoke TiUP while using the component.
+
+## Motivation
+
+- Some users do not like the mandatory update checks and additional outputs by TiUP before invoking the components.
+- For programs that are not only distributed through TiUP, the existing usage patterns have led to documentation fragmentation.
+
+## Detailed design
+
+1. Add `tiup link <component>[:version]` command to add soft link to $TIUP_HOME/bin
```

Comment:
Maybe something like this?
```
tiup link set <component> [version]
tiup link show [component] 
tiup link unset <component>
tiup link update <component> [version]
tiup link update <component> latest
```

Maybe use `update-alternatives` as a example:
```
[dvaneeden@dve-carbon ~]$ update-alternatives --list
nc                  	auto  	/usr/bin/ncat
mvn                 	auto  	/usr/share/maven/bin/mvn
man                 	manual	/usr/bin/man.man-db
man.7.gz            	manual	/usr/share/man/man7/man.man-pages.7.gz
soelim              	manual	/usr/bin/soelim.groff
gnuplot             	auto  	/usr/bin/gnuplot-qt
iptables            	auto  	/usr/sbin/iptables-legacy
ebtables            	auto  	/usr/sbin/ebtables-nft
arptables           	auto  	/usr/sbin/arptables-nft
xinputrc            	auto  	/etc/X11/xinit/xinput.d/ibus.conf
qtchooser-5         	auto  	/etc/xdg/qtchooser/5-64.conf
qtchooser-default   	auto  	/etc/xdg/qtchooser/5.conf
cifs-idmap-plugin   	auto  	/usr/lib64/cifs-utils/cifs_idmap_sss.so
ifup                	auto  	/usr/libexec/nm-ifup
cdrecord            	auto  	/usr/bin/xorrecord
mkisofs             	auto  	/usr/bin/genisoimage
cups_backend_smb    	auto  	/usr/bin/smbspool
libnssckbi.so.x86_64	auto  	/usr/lib64/pkcs11/p11-kit-trust.so
whois               	auto  	/usr/bin/jwhois
go                  	auto  	/usr/lib/golang/bin/go
llvm-config         	auto  	/usr/bin/llvm-config-64
llvm-config-16      	auto  	/usr/bin/llvm-config-64
jre_17              	auto  	/usr/lib/jvm/java-17-openjdk-17.0.8.0.7-1.fc38.x86_64
jre_17_openjdk      	auto  	/usr/lib/jvm/jre-17-openjdk-17.0.8.0.7-1.fc38.x86_64
java_sdk_17         	auto  	/usr/lib/jvm/java-17-openjdk-17.0.8.0.7-1.fc38.x86_64
java_sdk_17_openjdk 	auto  	/usr/lib/jvm/java-17-openjdk-17.0.8.0.7-1.fc38.x86_64
bytedance-feishu    	auto  	/usr/bin/bytedance-feishu-stable
print               	auto  	/usr/bin/lpr.cups
jre_openjdk         	auto  	/usr/lib/jvm/java-17-openjdk-17.0.8.0.7-1.fc38.x86_64
jre_11              	auto  	/usr/lib/jvm/java-11-openjdk-11.0.20.0.8-1.fc38.x86_64
jre_11_openjdk      	auto  	/usr/lib/jvm/jre-11-openjdk-11.0.20.0.8-1.fc38.x86_64
java_sdk_openjdk    	auto  	/usr/lib/jvm/java-17-openjdk-17.0.8.0.7-1.fc38.x86_64
java_sdk_11         	auto  	/usr/lib/jvm/java-11-openjdk-11.0.20.0.8-1.fc38.x86_64
java_sdk_11_openjdk 	auto  	/usr/lib/jvm/java-11-openjdk-11.0.20.0.8-1.fc38.x86_64
google-chrome       	auto  	/usr/bin/google-chrome-stable
java                	auto  	/usr/lib/jvm/java-17-openjdk-17.0.8.0.7-1.fc38.x86_64/bin/java
javac               	auto  	/usr/lib/jvm/java-17-openjdk-17.0.8.0.7-1.fc38.x86_64/bin/javac
ld                  	auto  	/usr/bin/ld.bfd
[dvaneeden@dve-carbon ~]$ update-alternatives --display ld
ld - status is auto.
 link currently points to /usr/bin/ld.bfd
/usr/bin/ld.bfd - priority 50
/usr/bin/ld.gold - priority 30
/usr/bin/ld.lld - priority 1
Current `best' version is /usr/bin/ld.bfd.
```

Reply:
actually, link and unlink are copied from homebrew